docs/NewProject.md

```diff
@@ -39,16 +39,28 @@ Select install_venv:
 Choose from 1, 2 [1]: 1
 ```
 
-Use the default for anything not specified. This command will take a moment to run.
+Use the default for anything not specified. This command will take a moment to run as it needs to clone the core F´ repository.
 
 The above command creates a new F´ project structure in a folder called `MyProject`, uses the `devel` branch of F´ as
 the basis for the project, and sets up the matching tools in a new Virtual Environment.
 
+We should navigate to the projects's directory and look around:
+
+```bash
+cd MyProject
+ls
+```
+This will show the following files:
+1. `fprime/`: F´ repository. Contains core F´ components, the API for the build system, among others
+2. `settings.ini`: allows users to set various settings to control the build
+3. `CMakeList.txt` and `project.cmake`: CMake files defining the build system
+4. `Components/`: directory to place user components in
+
 > Load the tools for this project via the virtual environment.
 > 
 > ```bash
-> cd MyProject
-> . venv/bin/activate
+> # In: MyProject
+> source venv/bin/activate
```

Comment:
We should be careful when using `source` instead of `.` as `.` is ironically more cross-shell than source.

Reply:
Done